[2025-07-02T11:03:13.513+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-02T11:03:13.551+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mysql_dag.create_table scheduled__2025-06-30T00:00:00+00:00 [queued]>
[2025-07-02T11:03:13.565+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mysql_dag.create_table scheduled__2025-06-30T00:00:00+00:00 [queued]>
[2025-07-02T11:03:13.566+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-07-02T11:03:13.585+0000] {taskinstance.py:2330} INFO - Executing <Task(MySqlOperator): create_table> on 2025-06-30 00:00:00+00:00
[2025-07-02T11:03:13.594+0000] {standard_task_runner.py:63} INFO - Started process 186 to run task
[2025-07-02T11:03:13.602+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'mysql_dag', 'create_table', 'scheduled__2025-06-30T00:00:00+00:00', '--job-id', '93', '--raw', '--subdir', 'DAGS_FOLDER/mysql.py', '--cfg-path', '/tmp/tmpce8cqc22']
[2025-07-02T11:03:13.612+0000] {standard_task_runner.py:91} INFO - Job 93: Subtask create_table
[2025-07-02T11:03:13.653+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-02T11:03:13.765+0000] {task_command.py:426} INFO - Running <TaskInstance: mysql_dag.create_table scheduled__2025-06-30T00:00:00+00:00 [running]> on host 76fc2b11611c
[2025-07-02T11:03:13.949+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Karthik' AIRFLOW_CTX_DAG_ID='mysql_dag' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2025-06-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-30T00:00:00+00:00'
[2025-07-02T11:03:13.951+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-02T11:03:13.995+0000] {sql.py:276} INFO - Executing: 
        create table if not exists books (
            id int auto_increment primary key,
            Date_Added datetime not null,
            Title varchar(1000) not null,
            Descriptions text,
            Category_Name varchar(1000),
            Star_Rating decimal(10,2),
            Price decimal(10,2) not null,
            Tax decimal(10,2),
            Price_with_Tax decimal(10,2) not null,
            Availability varchar(100),
            Availability_Count int,
            UPC  varchar(100) not null,
            Product_Type  varchar(100),
            Number_of_Reviews int,
            Image_URL  varchar(1000),
            Book_Link varchar(1000)
        )
        
[2025-07-02T11:03:14.008+0000] {base.py:84} INFO - Using connection ID 'mysql_conn' for task execution.
[2025-07-02T11:03:14.024+0000] {base.py:84} INFO - Using connection ID 'mysql_conn' for task execution.
[2025-07-02T11:03:14.107+0000] {sql.py:487} INFO - Running statement: 
        create table if not exists books (
            id int auto_increment primary key,
            Date_Added datetime not null,
            Title varchar(1000) not null,
            Descriptions text,
            Category_Name varchar(1000),
            Star_Rating decimal(10,2),
            Price decimal(10,2) not null,
            Tax decimal(10,2),
            Price_with_Tax decimal(10,2) not null,
            Availability varchar(100),
            Availability_Count int,
            UPC  varchar(100) not null,
            Product_Type  varchar(100),
            Number_of_Reviews int,
            Image_URL  varchar(1000),
            Book_Link varchar(1000)
        )
        , parameters: None
[2025-07-02T11:03:14.141+0000] {sql.py:496} INFO - Rows affected: 0
[2025-07-02T11:03:14.144+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-02T11:03:14.160+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=mysql_dag, task_id=create_table, run_id=scheduled__2025-06-30T00:00:00+00:00, execution_date=20250630T000000, start_date=20250702T110313, end_date=20250702T110314
[2025-07-02T11:03:14.218+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-02T11:03:14.233+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/models/baseoperator.py:1297 AirflowProviderDeprecationWarning: Call to deprecated class MySqlOperator. (Please use `***.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
[2025-07-02T11:03:14.244+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-02T11:03:14.248+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
